presto-main/src/test/java/com/facebook/presto/execution/MockRemoteTaskFactory.java

```diff
@@ -143,6 +145,11 @@ public MockRemoteTask createRemoteTask(
         return new MockRemoteTask(taskId, fragment, node.getNodeIdentifier(), executor, scheduledExecutor, initialSplits, totalPartitions, partitionedSplitCountTracker);
     }
 
+    @Override
+    public void removeRemoteSource(TaskId destinationTaskId, TaskId sourceTaskId, Consumer<PrestoException> onFailure)
```

Comment:
Do we want to do nothing or throw unsupported?

Reply:
Good question... I don't think today it will be called. So let's throw for now :)